docs/manager/graphql-reference/schema.graphql

```diff
@@ -1984,7 +1984,7 @@ type Mutations {
   clear_images(registry: String): ClearImages
 
   """Added in 25.4.0"""
-  purge_images(agent_id: String!, images: [ImageRefType]!): PurgeImages
+  purge_images(key: PurgeImagesKey!, options: PurgeImagesOptions): PurgeImages
```

Comment:
It's better to change the response type name from `PurgeImages` to `PurgeImagesPayload` or another name.